src/readers/reader.py

```diff
@@ -0,0 +1,28 @@
+import open3d as o3d
+from abc import ABC, abstractmethod
+from nptyping import NDArray, Shape, Float, UInt8
+from src.database import Database
+
+
+class Reader(ABC):
+    @abstractmethod
+    def read_dataset(self) -> Database:
+        rgb_images, pcds, trajectory = self._get_images_pcds_traj()
```

Comment:
Why is it mixed in one method?

Reply:
because we should have traj, pcds and rgbs with equal len